core/domain/change_domain.py

```diff
@@ -164,6 +179,11 @@ def validate_dict(self, change_dict):
                 valid_cmd_attribute_specs = copy.deepcopy(cmd)
                 break
 
+        for cmd in self.DEPRECATED_COMMANDS:
+            if cmd == cmd_name:
+                raise utils.DeprecatedError(
+                    'Command %s is deprecated' % cmd_name)
```

Comment:
Should we raise here? Could this lead to some problems in the production?

Reply:
I tired to raise an error here because, these commands are deprecated. So this will raise an exception if a deprecated command is used in the production whereas the validation jobs will not flag the error, since we are in the process of removing them. Originally if this change is not there, then these deprecated commands will still raise an exception. So this will not cause any new problems in the production whereas it will solve the errors being raised in the validation jobs.